src/handle-checkout-mutation.js

```diff
@@ -1,8 +1,14 @@
 export default function handleCheckoutMutation(mutationRootKey, client) {
-  return function({data, model}) {
+  return function({data, errors, model}) {
     const rootData = data[mutationRootKey];
     const rootModel = model[mutationRootKey];
-    const userErrors = rootData.userErrors;
+    let userErrors = [];
+
+    if (errors != null && errors.length) {
+      userErrors = errors;
```

Comment:
just call reject in each branch. no need to assign names.

`if (errors && errors.length) {
  return Promise.reject(...) {
} else if (rootData && rootData.userErrors && rootData.userErrors.length)
  return Promise.reject(...) {
}